crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -306,6 +309,14 @@ impl ModuleLoader {
         }
       }
       self.remaining -= 1;
+
+      let mut count = load_module_rx_guard.len();
+      while count > 0 {
```

Comment:
Here the channel can't starting recevier msg because the `await this.load` is block the `ModuleLoader#channel`. I will try to merge the two channel using one channel.